tests/lib.rs

```diff
@@ -47,6 +47,30 @@ fn new_mock_client_with_unsafe_chars() -> BasicClient {
     )
 }
 
+#[test]
```

Comment:
nice tests! mind adding a simple one for `exchange_code_pkce` (maybe similar to `test_exchange_code_successful_with_minimal_json_response`)?